web/src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java

```diff
@@ -947,7 +985,20 @@ private Object validateStudyEvent(StudyEventDataBean studyEventDataBean, StudySu
     }
 
 
-    public ErrorObj validateStartAndEndDateAndOrder(StudyEventDataBean studyEventDataBean) {
+    private boolean isEventCrfCompleted(EventCrf eventCrf) {
+    	if(eventCrf.getStatusId() == EventCRFStatus.INITIAL_DATA_ENTRY_COMPLETE.getCode()) {
+    		return true;
+    	}else if(eventCrf.getStatusId() == EventCRFStatus.DOUBLE_DATA_ENTRY_COMPLETE.getCode()) {
+    		return true;
+    	}else if(eventCrf.getStatusId() == Status.UNAVAILABLE.getCode()) {
+    		return true;
```

Comment:
Status.UNAVAILABLE for event crf means more than crf is complete. It could be removed, locked, ...
If this method checks whether or not an event_crf is complete only, this should be removed from here.

Reply:
as we discussed, we changed the code to only check Status.UNAVAILABLE
and there is another ticket https://jira.openclinica.com/browse/OC-11487
to track the existing issue.